api/README.md

```diff
@@ -164,11 +166,13 @@ automatic sign-in experience.
 
 ```java
 // Craft the valid credential.
-String identifer = "joe@gmail.com";
+CredentialClient client = CredentialClient.getInstance(getContext());
```

Comment:
Nit: moving the client creation above the ```// Craft the valid credential``` comment helps visually separate the two operations and matches the account creation example.

Reply:
Fixed